addons/grass_force/functions/fnc_loop.sqf

```diff
@@ -0,0 +1,24 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function checks if grass should be forced, enables it and schedules next check.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+
+if (!GVAR(enabled)) exitWith { nil };
```

Comment:
Empty code block is `nil` already. If you're not executing anything inside it's fine to leave it empty.
```suggestion
if (!GVAR(enabled)) exitWith {};
```